lua/ulx/modules/sh/cfc_propify.lua

```diff
@@ -153,6 +153,8 @@ function cmd.propifyTargets( caller, targets, modelPath, shouldUnpropify, overri
                 ULib.tsayError( caller, ulx.getExclusive( ply, caller ), true )
             elseif not ply:Alive() then
                 ULib.tsayError( caller, ply:Nick() .. " is dead and cannot be propified!", true )
+            elseif ( ply.IsInPvp and ply:IsInPvp() ) and not caller:IsAdmin() then
```

Comment:
This is going to prevent moderators from using the command on pvpers, im not sure thats desired.